src/libm-tester/iutsimdmain.c

```diff
@@ -6,44 +6,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdint.h>
-#include <signal.h>
-#include <setjmp.h>
-
-static jmp_buf sigjmp;
+#include "check_feature.h"
 
 int do_test(int argc, char **argv);
-void check_featureDP();
-void check_featureSP();
-
-static void sighandler(int signum) {
```

Comment:
@shibatch Do you have any idea why by moving this SIGILL handler in a separate module the unsupported FMA4  iut tests started failing? See https://travis-ci.org/shibatch/sleef/jobs/286683373#L741